modules/ocm-overview-tab.adoc

```diff
@@ -21,6 +21,9 @@ endif::openshift-rosa[]
 ifdef::openshift-rosa-hcp[]
 * **Availability** shows multizone for {rosa-short} clusters.
 endif::openshift-rosa-hcp[]
+ifndef::openshift-enterprise[]
+*  **Channel group** shows the update channel for the cluster, such as stable,fast, or eus. Support for channel groups varies by cluster version. If support channel editing is available, this field can be changed by clicking the pencil icon.
```

Comment:
we mention only stable and eus https://docs.redhat.com/en/documentation/red_hat_openshift_service_on_aws/4/html/cli_tools/rosa-cli, I don't know if we want to align the list